content/js/record.md

```diff
@@ -309,23 +314,18 @@ skygear.publicDB.query(new skygear.Query(Note))
   .then((records) => console.log(records[0]));
 ```
 
-```
-RecordCls {
-  $transient: (...)
-  _id: "3b9f8f98-f993-4e1d-81c3-a451e483306b"
-  _recordType: "note"
-  _transient: Object
-  access: (...)
-  attributeKeys: (...)
-  createdAt: Thu Jul 07 2016 12:12:42 GMT+0800 (CST)
+``` javascript
+/* Type: RecordCls */ {
+  createdAt: new Date("Thu Jul 07 2016 12:12:42 GMT+0800 (CST)")
+  updatedAt: new Date("Thu Jul 07 2016 12:42:17 GMT+0800 (CST)")
   createdBy: "118e0217-ffda-49b4-8564-c6c9573259bb"
-  id: "note/3b9f8f98-f993-4e1d-81c3-a451e483306b"
-  ownerID: "118e0217-ffda-49b4-8564-c6c9573259bb"
-  recordType: (...)
-  updatedAt: Thu Jul 07 2016 12:42:17 GMT+0800 (CST)
   updatedBy: "118e0217-ffda-49b4-8564-c6c9573259bb"
+  ownerID: "118e0217-ffda-49b4-8564-c6c9573259bb"
+  id: "note/3b9f8f98-f993-4e1d-81c3-a451e483306b"
+  _id: "3b9f8f98-f993-4e1d-81c3-a451e483306b"
+  recordType: "note"
 }
 ```
 
-Please read the [above](#record) section for more about `_id`. Check the server
-[database schema](/server/guide/database-schema) page for more column names.
+Check the server [database schema](/server/guide/database-schema) page
+for more column names.
```

Comment:
simply "for more information"? I am not exactly sure what "more column names" refer to in this context